Makefile

```diff
@@ -0,0 +1,16 @@
+all: test
+
+install-ci:
+	pip install -r requirements/ci.txt
+
+install-local:
+	pip install -r requirements/local.txt
```

Comment:
I am using [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), so my virtualenv is not on venv/

You can add a make task to activate virtualenv, anyways, if you want.